src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="big_form">
+        <img
+        src="./images/Search.svg"
+        alt="search-icon"
+        class="icon-search-big"
+        >
```

Comment:
Also fix other places all over the page
```suggestion
        <img
          src="./images/Search.svg"
          alt="search-icon"
          class="icon-search-big"
        >
```